index.html

```diff
@@ -53,35 +231,133 @@ <h3>LET’S CONNECT</h3>
               <img
                 class="social linkedin"
                 src="image/icons/linkedin.png"
-                alt="linkedin icon"
+                alt="Twitter icon"
               />
             </a>
           </li>
           <li>
             <a href="#">
-              <img class="social m" src="image/icons/m.png" alt="m icon" />
+              <img class="social fin" src="image/icons/fin.png" alt="Twitter icon" />
             </a>
           </li>
           <li>
             <a href="#">
               <img
-                class="social github"
-                src="image/icons/github.png"
-                alt="github icon"
+                class="social twitter"
+                src="image/icons/twitter.png"
+                alt="Twitter icon"
               />
             </a>
           </li>
           <li>
             <a href="#">
               <img
-                class="social fin"
-                src="image/icons/fin.png"
-                alt="fin icon"
+                class="social m"
+                src="image/icons/m.png"
+                alt="Twitter icon"
               />
             </a>
           </li>
         </ul>
+        <div>
+        <button type="button" class="button button-enabled">
+          See my resume
+        </button>
+        </div>
+        <div class="other-info">
+          <ul class="flex other-info-wrap padding-0">
+            <li class="other-info-list">
+              <div class="flex li-flex wrap">
+                <h4>Languages</h4>
+                <span>
+                  <svg
+                    width="24"
+                    height="24"
+                    viewBox="0 0 24 24"
+                    fill="none"
```

Comment:
- [ ] the language and drop down icon should be on same line.
![Screenshot (65)_LI](https://user-images.githubusercontent.com/65298048/170499652-1a4fc09c-2bdc-4184-ad9f-7719810776b2.jpg)